src/__tests__/utils.test.ts

```
@@ -0,0 +1,72 @@
+/* eslint-env jest */
+
+import { getConfig, isApiCompatible, isApiAvailable } from '../utils.js'
+import type { APIHealthResponse } from '../types.js'
+
+const originalEnv = process.env.VISIONBOARD_INSTANCE_URL
+let mockResponse: APIHealthResponse
+
+beforeEach(() => {
+  delete process.env.VISIONBOARD_INSTANCE_URL
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)